tests/test_sensor_suite.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os
+
+# Sort the paths out to run from this file
+parent_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+src_path = os.path.join(os.path.sep,parent_path, 'src')
+sys.path.append(src_path)
+
+from regressors import RBFModel, LModel, GPModel, CSModel, CTModel, PModel
+from sensor_group import SensorSuite, SymmetryManager
+from fields import ScalarField, VectorField
+from sensors import Sensor
+import numpy as np
+import unittest
+
+
+# Key
+# g means grid test (2D field plane)
+# l means line test (1D field line)
+
+
+class Test(unittest.TestCase):
+    def test_sensor_suite1(self):
+        def f(x): return np.zeros(x.shape)
+        sensor = Sensor(
+            0,
+            f,
+            0.1,
+            np.array([[-50], [50]]),
+            np.array([[0, 0], [-1, -1], [1, 1]])
+        )
+
```

Comment:
Suggestion: consider using a lambda function as an anonymous shorthand way to pass this function arg, this avoids the confusing `def` line within a test, which could be misread at a glance as another test. You may need to check if they way I've suggested implementing it actually works, but I think this is how lambda functions work.
```suggestion
        sensor = Sensor(
            0,
            lambda x : np.zeros(x.shape),
            0.1,
            np.array([[-50], [50]]),
            np.array([[0, 0], [-1, -1], [1, 1]])
        )

```